File: 250316/합과 차의 나눗셈/divide-of-sum-and-sub.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {

    int a = 0;
    int b = 0;

    cin >> a >> b;

    int sum = a + b;
    int minus = a - b;

    double total = (double) sum / minus;

    cout << fixed;
    cout.precision(2);

    cout << total;
}

File: 250316/합과 차의 나눗셈/divide-of-sum-and-sub.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int a = sc.nextInt();
        int b = sc.nextInt();

        int sum = a + b;
        int minus = a - b;

        double total = (double) sum / minus;

        System.out.printf("%.2f", total);
    }
}